spec/models/notification_spec.rb

```diff
@@ -272,24 +272,22 @@
     end
 
     context "when reactable is receiving notifications" do
-      it "sends a notification to the author of a comment" do
-        reaction = create(:reaction, reactable: comment, user: user)
+      let(:reaction) { create(:reaction, reactable: comment, user: user) }
+      let(:count) { comment.user.notifications.count }
 
-        perform_enqueued_jobs do
-          expect do
-            described_class.send_reaction_notification(reaction, reaction.reactable.user)
-          end.to change(comment.user.notifications, :count).by(1)
+      before do
+        @previous_notification_count = comment.user.notifications.count
```

Comment:
we shouldn't use instance variables, did you have any issues keeping the style of the specs as is? let me know